src/Theme.re

```diff
@@ -64,8 +64,9 @@ module Helpers = {
 module Bases = {
   let input = (~theme) =>
     style([
-      padding2(~v=rem(0.75), ~h=rem(0.9)),
+      padding2(~v=px(0), ~h=px(12)),
```

Comment:
shoulnd't this be the predetermined constants to ensure it's matching the grid?

Reply:
All inputs are set to have `height: 40px` that will match the grid. The inside horizontal padding won't affect the grid.